### PR TITLE
Added homepage header files

### PR DESCRIPTION
**Pull request by sandhana14**
# Description

Created homepage header component which has a title and robot image

Fixes # (issue)
#4 

# How to test?

npm run storybook

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by abedaarabi — APPROVED** [review]
I think it looks great:
image

File: src/client/components/HomepageHeader/Header.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    </header>
+  );
+}
+
+Header.propTypes = {
+  color: PropTypes.string.isRequired,
+  size: PropTypes.number.isRequired,
+};
+
+export default Header;
```
Comment (TITHI1244) [inline src/client/components/HomepageHeader/Header.js:22]: The Header looks super nice to me :) @sandhana14.
But I wonder, is it necessary to use props as I was thinking we would not need to change the header color or font-size somewhere in the app. Rather it should be static.

**Review by TITHI1244 — COMMENTED** [review]
The code and design looks perfect :)

File: src/client/components/HomepageHeader/Header.js (hunk 1 above)
Comment (sandhana14) [reply src/client/components/HomepageHeader/Header.js:22]: Hi Tithi. Thank you for your review. I felt the same but I wanted to give some flexibility that's why I tried.

**Comment by sandhana14** [comment]
> I think it looks great:
> image

Thank you

**Comment by cph-kiwi** [comment]
This looks good and seems ready for approval and merging.

**Review by cph-kiwi — APPROVED** [review]

**Review by TITHI1244 — APPROVED** [review]
Everything looks perfect

File: src/client/components/HomepageHeader/Header.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import robot from '../../assets/images/robot.png';
+import './header.css';
+import PropTypes from 'prop-types';
+
+function Header({ color, size }) {
```
Comment (bytewiz) [inline src/client/components/HomepageHeader/Header.js:6]: Could consider naming `size` to `fontSize` instead as it is the fontSize you change with this an not the size of the header :)

**Review by bytewiz — APPROVED** [review]
Seems good just one comment on the name for the `size` prop in `Header`

**Pull request merged**